### PR TITLE
cli: rename `contract compile` --debug flag

### PR DESCRIPTION
**Pull request by fyrchik**
- Rename it to `--verbose`, which makes more sense.
- Fix a bug where hex was outputted irregardless of if `--verbose` was specified.

### CODE REVIEW

File: cli/smartcontract/smart_contract.go [hunk 1]
```diff
@@ -347,14 +347,15 @@ func contractCompile(ctx *cli.Context) error {
 
 	o := &compiler.Options{
 		Outfile: ctx.String("out"),
-		Debug:   ctx.Bool("verbose"),
```
Comment (roman-khimov) [inline cli/smartcontract/smart_contract.go]: Well, maybe we should remove `Debug` from options at the same time? It's not used other than here.

Comment (fyrchik) [reply cli/smartcontract/smart_contract.go]: Done.

File: cli/smartcontract/smart_contract.go [hunk 2]
```diff
@@ -89,8 +89,8 @@ func NewCommands() []cli.Command {
 						Usage: "Output of the compiled contract",
 					},
 					cli.BoolFlag{
-						Name:  "debug, d",
-						Usage: "Debug mode will print out additional information after a compiling",
+						Name:  "verbose, v",
+						Usage: "Print out additional information after a compiling",
 					},
 					cli.StringFlag{
 						Name:  "emitdebug",
```
Comment (roman-khimov) [inline cli/smartcontract/smart_contract.go:96]: BTW, this whole change was intended to free `debug, d` for this option, so why isn't it used here?

Comment (roman-khimov) [reply cli/smartcontract/smart_contract.go:96]: ping

Comment (fyrchik) [reply cli/smartcontract/smart_contract.go:96]: Done.

File: cli/smartcontract/smart_contract.go [hunk 3]
```diff
@@ -89,11 +89,11 @@ func NewCommands() []cli.Command {
...
 					},
 					cli.BoolFlag{
-						Name:  "debug, d",
-						Usage: "Debug mode will print out additional information after a compiling",
+						Name:  "verbose, v",
+						Usage: "Print out additional information after a compiling",
 					},
 					cli.StringFlag{
-						Name:  "emitdebug",
+						Name:  "debug",
```
Comment (roman-khimov) [inline cli/smartcontract/smart_contract.go:96]: `debug, d`, let us use the short form, please.

**Pull request merged**